packages/esm-patient-labs-app/src/lab-orders/add-lab-order/add-lab-order.workspace.tsx

```diff
@@ -30,7 +30,7 @@ export default function AddLabOrderWorkspace({
 }: AddLabOrderWorkspace) {
   const { t } = useTranslation();
 
-  const { patient, isLoading: isLoadingPatient } = usePatient();
+  const { patient, isLoading: isLoadingPatient, patientUuid } = usePatient();
```

Comment:
All workspaces should receive the `patientUuid` prop [implicitly](https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/packages/esm-patient-common-lib/src/types/workspace.ts#L22). You shouldn't need to drill the prop down to `LabOrderForm`.

Reply:
Found a weird behaviour testing this that's why i added this change. Maybe it was a unique case.